### ISSUE TITLE
[BUG]: The A11y tool looks for label/title for an element with role=presentation

### ISSUE THREAD

**Issue by Mahalakshmivignesh** [IBMa/equal-access#2060]
### Project

other

### Browser

Chrome

### Operating system

MacOS

### Description

I was having issues with the 'SVG element has no accessible name error' from the accessibility checker tool. as per the advice, i have taken up the issue with the carbon team and the fix for this is done in the latest charts version. This helped in getting rid of the above error in most of the places. But even after using the latest version of charts, i am still facing the error when the role of the element is 'presentation'. As per the developer's comment, this is a potential bug in the checker tool. Error screenshot 
![image](https://github.com/user-attachments/assets/e2cdf55f-7ba4-4657-88a1-fbc0fa9fffc9)
please let me know if you need any more details

### Steps to reproduce

Create an svg component with role='presentation'. ideally i shouldnt be seeing the 'The SVG element has no accessible name’' error.
